mark_broken.py

```diff
@@ -65,6 +77,34 @@ def mark_broken():
     finally:
         os.remove(token_path)
 
+    with tempfile.TemporaryDirectory() as tmpdir:
+        subprocess.check_call(
+            "git clone https://github.com/conda-forge/"
+            "conda-forge-repodata-patches-feedstock.git",
+            cwd=tmpdir,
+            shell=True,
+        )
+
+        subprocess.check_call(
+            "git remote set-url --push origin "
+            "https://%s@github.com/conda-forge/"
+            "conda-forge-repodata-patches-feedstock.git" % os.environ["GITHUB_TOKEN"],
+            cwd=os.path.join(tmpdir, "conda-forge-repodata-patches-feedstock"),
+            shell=True
+        )
+
+        subprocess.check_call(
+            "git commit --allow-empty -am 'resync repo data for broken packages'",
+            cwd=os.path.join(tmpdir, "conda-forge-repodata-patches-feedstock"),
+            shell=True
+        )
+
+        subprocess.check_call(
```

Comment:
If check call fails, the token will be printed to stdout right?

Reply:
On many ci services the tokens get masked by the ci service itself. GitHub redacts secrets in logs.

Reply:
Quote from the docs 

> To ensure that GitHub redacts your secret in logs, avoid using structured data as the values of secrets. For example, avoid creating secrets that contain JSON or encoded Git blobs.